R/defaults.R

```diff
@@ -75,7 +75,7 @@ epidist_stancode.default <- function(data, ...) {
 #' @method epidist default
 #' @family defaults
 #' @export
-epidist.default <- function(data, formula = list(mu ~ 1, sigma ~ 1),
+epidist.default <- function(data, formula = brms::bf(mu ~ 1, sigma ~ 1),
```

Comment:
can we entirely avoid specifying sigma and thus make this portable across all families? I think it should build down to the same thing?

Reply:
Yes good point. Not sure. I'll try it and report back

Reply:
No it doesn't work (breaks 7 tests)

Reply:
I think this is solvable so will make an issue for it

Reply:
https://github.com/epinowcast/epidist/issues/222